### ISSUE TITLE
multi level finished-when

### ISSUE THREAD

**Issue by lazytesting** [arguseyes/argus-eyes#12]
Currently finished-when is only implemented at project level. 
In some scenarios it can be hard to write a check which covers all scenarios.

That's why it's useful to have to be able to implement specific finished-when checks also per page and per component.

**Comment by branneman** [comment]
Great idea, but when we have this, do we still need the global one?

```js
{
  "sizes": ["320x480", "1280x768"],
  "finished-when": "argus-eyes-scripts/homepage.js", // <-- already possible
  "pages": [ {
      "name": "homepage",
      "url": "http://localhost:3000/",
      "components": [ "navigation", "news-items" ],
      "finished-when": "argus-eyes-scripts/homepage.js" // <-- to be added
    }, {
      "name": "contact",
      "url": "http://localhost:3000/contact.html",
      "components": [ "navigation" ]
    } ],
  "components": [ {
      "name": "navigation",
      "selector": ".nav",
      "finished-when": "argus-eyes-scripts/navigation.js" // <-- to be added
    }, {
      "name": "news-items",
      "selector": ".news-items",
      "ignore": [ ".last-updated" ]
    } ]
}
```

**Comment by branneman** [comment]
Fixed in 764377010e0006e21716c53539ab910bce17007d, will be shipped with the upcoming 0.6.0
